lib/AllocateDescriptorsPass.cpp

```diff
@@ -160,8 +160,8 @@ class AllocateDescriptorsPass final : public ModulePass {
 } // namespace
 
 char AllocateDescriptorsPass::ID = 0;
-static RegisterPass<AllocateDescriptorsPass> X("AllocateDescriptorsPass",
-                                               "Allocate resource descriptors");
+INITIALIZE_PASS(AllocateDescriptorsPass, "AllocateDescriptorsPass",
+                "Allocate resource descriptors", false, false)
```

Comment:
One of the falses is whether this is an analysis. What is the other?

Reply:
Whether it's a CFG-only pass.